### ISSUE TITLE
losses_pytorch vs test/loss_functions

### ISSUE THREAD

**Issue by daandres** [JunMa11/SegLoss#20]
Hi, 

thanks for this great collection of segmentation losses. 
But I wonder, what are the differences between `losses_pytorch` (https://github.com/JunMa11/SegLoss/tree/master/test/loss_functions) and `test/loss_functions` (https://github.com/JunMa11/SegLoss/tree/master/test/loss_functions)?
And which one would recommend for custom test? 
I saw that `test/loss_functions`have dependencies to nnU-Net, while the others don't. 

Thank you

**Comment by JunMa11** [comment]
Hi @daandres,

Thanks for your interest.
- If you want to use the loss functions in your own network, I would recommend  https://github.com/JunMa11/SegLoss/tree/master/losses_pytorch
- On the other hand, I tested all the loss functions with nnU-Net V1 (https://github.com/JunMa11/SegLoss/tree/master/test) .

Now, nnU-Net has moved to V2 but I have not created a new version for the loss functions. If you want to use nnU-Net V2 with different loss functions, I would recommend https://github.com/MIC-DKFZ/nnUNet/tree/master/nnunet/training/network_training/nnUNet_variants/loss_function

**Comment by daandres** [comment]
thank you @JunMa11 for clarification and the additional repo.
